Added Galaxy Nexus skin to repo.

aaronsnoswell opened the pull request:
Hi there! I've added a Galaxy Nexus skin to this repo, as well as updated the readme a bit. I've included a res directory where the Gimp file for the Galaxy Nexus is stored - in case we need to make changes later.

Commit:
Add Galaxy Nexus Emulator Skin.

Commit:
Update Readme.

Commit:
Add Gimp file for Galaxy Nexus.

HBehrens commented:
That's great, Aaron. I will try it out and will merge your pull request if everythings runs smoothly. Are there any credits you have to give for the graphics?

aaronsnoswell commented:
Sweet. No pressure :) I had a look at the DeviantArt guy who you sourced yours from, but he's since started charging for his .psd's (fair enough!) So no, I don't - the Galaxy Nexus image I based everything off was simply one of the press release ones.

HBehrens commented:
Meanwhile, I had a detailed look at your pull request. The graphics are well done, good job! It's cool to have yet another phone on the list :) I encountered some issues, though:
- in landscape mode: the screen is not placed correctly (above case)
- in landscape mode: highlighted state of back and menu buttons are swapped
- the overlay is not placed correctly (too far to the bottom and right)

Can you fix this?

Also, to keep the repo as small as possible I'd suggest we put your gimp file and my PSDs into a submodule. What do you think?

Commit:
Fix back and menu buttons in landscape.

Commit:
Fix Overlay positioning.

Commit:
Extend overlay to cover notification bar as well

Commit:
Update Gimp file.

Commit:
Fix screen position in landscape.

I don't know why, but for some reason in landscape mode, the emulator
expects the coords of the BOTTOM left corner. Surely this is a bug?

aaronsnoswell commented:
Thanks for digging up these issues - I hadn't even thought of checking it in landscape. And thanks for the compliment :) These are all fixed now.

Can you clarify what you mean by keeping the PSD's and XCF's in a submodule - another git repo?

aravance commented:
A little input, for the Galaxy Nexus, there should be no hardware buttons. The software buttons of 4.0.3 should be all there is. The screen should probably extend down to where the current hardware buttons are located.

Apart from that great work! It's nice being able to work on an emulator that looks like the actual device.

aaronsnoswell commented:
Ha - Can't believe I didn't even think of that! Doh. Next time I'm using this, I'll update it to NOT have h/w buttons! Cheers :P
